packages/mermaid-zenuml/src/detector.ts

```diff
@@ -0,0 +1,12 @@
+export const id = 'zenuml';
+
+export const regexp = /^\s*zenuml/;
+
+export const detector = (txt: string) => {
+  return txt.match(regexp) !== null;
+};
+
+export const loadDiagram = async () => {
+  const { diagram } = await import('./diagram-definition');
+  return { id, diagram };
+};
```

Comment:
Is `master` branch ahead of `develop` branch? Shall we pull request to master or develop branch? I am asking because develop branch has not been changed.

Reply:
Yeah, we're not in a clean state at the moment. Will be done soone. 

Master is ahead of develop.

I pointed this out as you'll have to change this once we complete the merge.